src/semantic-ui/site/elements/list.overrides

```diff
@@ -32,6 +32,10 @@ List Overrides - REACT-INVENIO-APP-ILS
     padding: 0.5em;
   }
 
+  &.metadata-sharp{
```

Comment:
To be discussed if bold+black is enough. In any case, instead of `sharp`, why not creating a class `.color-black` that can be eventually re-used?